docs/output.md

```diff
@@ -14,50 +10,197 @@ The directories listed below will be created in the results directory after the
+  - `entities_proteins.tsv`: matches proteins to entities. Contains entity_id and protein_id for all unique entity - protein combinations.
+  - `peptides.tsv.gz`: contains peptide_id and peptide_sequence for all unique peptides. Peptides are generated for downloaded or predicted proteins.
+  - `proteins_peptides.tsv`: matches peptides to proteins. Contains protein_id, peptide_id and count (number of occurences of peptide in respective protein) for all unique protein - peptide combinations.
+  - `predictions.tsv.gz`: contains peptide_id, prediction_score (epitope prediction score) and allele_id for all unique peptide - allele combinations.
+
+</details>
+
+Additionally the pipeline reports some statistics on protein and peptide numbers.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `db_tables/`
+  - `stats.txt`: contains statistics: unique protein counts, total peptide counts, unique peptide counts, unique peptides across all conditions.
 
-* [FastQC](#fastqc) - Read quality control
-* [MultiQC](#multiqc) - Aggregate report describing results from the whole pipeline
-* [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+</details>
 
-## FastQC
+The epitope prediction scoring system and resulting prediction scores depend on the chosen prediction method (`--pred_method`).
 
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences.
+- [SYFPEITHI](http://www.syfpeithi.de/index.html): In SYFPEITHI the scoring is based on a position-specific scoring matrix (PSSM), which is built using known MHC binders. For each allele another matrix is used. For comparability the scores are normalized to the highest scoring ligand of each allele, resulting in scores between 0 and 1. Therefore, the final score represents how well a peptide is able to bind in comparison to all annotated binders of a specific allele. For downstream processes such as `prepare_entity_binding_ratios` peptides with a score of ≥0.5 are classified as binders.
+- [MHCflurry](https://pubmed.ncbi.nlm.nih.gov/29960884/) and [MCHnuggets](https://pubmed.ncbi.nlm.nih.gov/31871119/): The score is based on a affinity scoring representing an IC<sub>50</sub> (the concentration that leads to 50% inhibition of the binding of a standard reference peptide, note that lower IC<sub>50</sub> values indicate higher binding affinities). The affinity score is then log-transformed and converted to a scale from 0 to 1 by using the formula: <code>1-log<sub>50000</sub>(affinity score)</code>. For downstream processes such as `prepare_entity_binding_ratios` a threshold of ≥0.426 is used to classify binders, corresponding to an IC<sub>50</sub> of ≤500.
```

Comment:
Okay I guess the text is not informative enough then! That's basically the equivalent score (calculated IC50 by epytope) of an affinity of <= 500nM
The documentation of MHCFlurry states: 
> A commonly-used threshold for peptides with a reasonable chance of being immunogenic is 500 nM.

Do you think it would help to attach a citation here?

Reply:
And 0.426 is the "raw" prediction score? If yes I think it's already enough to state this